button.css

```diff
@@ -0,0 +1,88 @@
+:root {
+  font-size: 62.5%;
+  box-sizing: border-box;
+}
+.button-container {
+  padding: 1rem 2rem;
+  margin: 1rem;
+  /* margin-left:-0.3rem; */
+  display: inline-block;
+  font-weight: 800;
+  font-size: 1.8rem;
+  border: none;
+  border-radius: 1rem 1rem 1rem 1rem;
+
+  color: black;
+  font-family: "roboto";
+}
+.primary-button {
+  background-color: #49a0eb;
```

Comment:
use var for color scheme, it will be easier to modify and reuse

Reply:
+1

Reply:
made changes and used color schemes